Norm_end_to_end.Rmd

```diff
@@ -395,35 +451,71 @@ intern_standard$CV_NOMIS <- rowRsd(
 
 cpt <- paste("Internal standards with detected and assigned features coefficient
              of variation before and after different method of normalization.")
-pandoc.table(intern_standard[, c("abbreviation", "CV_raw", "CV_MRM", 
-                                 "CV_MRM_500", "CV_ruvr", 
+pandoc.table(intern_standard[, c("abbreviation", "CV_raw", "CV_MRM",
+                                 "CV_MRM_500", "CV_ruvr",
                                  "CV_ruv3", "CV_NOMIS")],
              style = "rmarkdown", caption = cpt)
 ```
+
 At first glance it seems that MRM 500 works better than MRM. and RUV random
-completely remove variation.. .
+completely remove variation.. . RUV and NOMIS used however the internal
```

Comment:
just wanted to make sure that we are not overestimating the CV because (maybe?) IS was added at different steps in the sample prep.